core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -251,6 +252,11 @@ public static Set<Integer> getIntegerSet(String property, JsonNode node) {
         .build();
   }
 
+  public static List<Long> getLongList(String property, JsonNode node) {
```

Comment:
this is to align the existing code as there's a `getIntegerList()` but no `getLongList()` method

Reply:
Is this used anywhere?

Reply:
currently it's only used in the test itself